src/bin/compute-confidence-interval-wer.cc

```diff
@@ -0,0 +1,138 @@
+// bin/compute-confidence-interval-wer.cc
+
+// Copyright 2009-2011  Microsoft Corporation
+//                2014  Johns Hopkins University (authors: Jan Trmal, Daniel Povey)
+//                2015  Brno Universiry of technology (author: Karel Vesely)
+//                2016  Nico
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#include "base/kaldi-common.h"
+#include "util/common-utils.h"
+#include "util/parse-options.h"
+#include "tree/context-dep.h"
+#include "util/edit-distance.h"
+#include "base/kaldi-math.h"
+
+int main(int argc, char *argv[]) {
+  using namespace kaldi;
+  typedef kaldi::int32 int32;
+
+  try {
+    const char *usage =
+        "Compute a bootstrapping of WER to extract the 95\% confidence interval\n"
+        "Takes two transcription files, in integer or text format,\n"
+        "and outputs overall WER statistics to standard output along with its confidence interval"
+        "supposing that WER of sentences subsamples follow the normal distribution.\n"
```

Comment:
Rather than saying supposing that WER of sentences subsamples follow the normal distribution, say
"using the 'bootstrap' method of Bisani and Ney\n"

Don't say 'Takes two transcription files, in integer or text format'
because it implies you are comparing the accuracy of two independent transcriptions.
Say
 'Takes a reference and a transcription file, in integer or text format.\n'

It would be better if you also implemented the two-system comparison method that they described in that paper-- you could implement it in the same binary, so if you give an extra argument it does the two-system one.
I suggest to call the binary compute-wer-confidence, so that it's more easily discoverable once you know about compute-wer.